fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -317,10 +317,14 @@ private void initProfile(long beginTimeInNanoSecond) {
             if (coord.getQueryProfile() != null) {
                 coord.getQueryProfile().getCounterTotalTime()
                         .setValue(TimeUtils.getEstimatedTime(beginTimeInNanoSecond));
+                long profileCollectStartTime = System.currentTimeMillis();
                 coord.endProfile();
```

Comment:
What's the different between the endTime here with endtime in `writeProfile`

Reply:
Before: the time contains two parts: part one is the time usage for waiting for all instances to report profile(i.d. `endProfile`), the second is the profile merge time usage(i.e. `buildMergedQueryProfile`).

After: the time only contains the first part.